docs/content/preview/drivers-orms/_index.md

```diff
@@ -24,6 +24,12 @@ In addition to the compatible upstream PostgreSQL drivers, YugabyteDB also suppo
 
 All YugabyteDB smart driver libraries are actively maintained, and receive bug fixes, performance enhancements, and security patches.
 
+{{< note title="Note" >}}
+
+To take advantage of smart driver load balancing features when connecting to clusters in YugabyteDB Managed, the smart driver and application must be deployed in a VPC that has been peered with the cluster VPC. For information on VPC networking in YugabyteDB Managed, refer to [VPC network](../yugabyte-cloud/cloud-basics/cloud-vpcs/).
```

Comment:
Instead of this:

> the smart driver and application must be deployed in a VPC that has been peered with the cluster VPC

More appropriate will be:

> Applications using smart drivers must be deployed in a VPC that has been peered with the cluster VPC

The reason is that drivers are not explicitly deployed but they are kind of embedded in the applications which use them.

Reply:
Additionally we can also write:

If applications are accessing the clusters created by YugabyteDB Managed from some other network, it is advisable to use the upstream PostgreSQL drivers and depend on the load balancer for the cluster. If the applications use smart drivers even in this case also then the smart driver falls back to the upstream driver behaviour.